clients/js/README.md

```diff
@@ -28,16 +28,14 @@ Commands:
   worm edit-vaa                      Edits or generates a VAA
   worm evm                           EVM utilities
   worm generate                      generate VAAs (devnet and testnet only)
-  worm info                          Contract, chain, rpc and address informatio
-                                     n utilities
+  worm info                          Contract, chain, rpc and address information utilities
```

Comment:
I think this might be auto generated so modifying it might make the generation check fail.